src/microbe_stage/gui/ExtinctionBox.cs

```diff
@@ -11,9 +12,13 @@ public partial class ExtinctionBox : CustomWindow
     [Export]
     public NodePath LoadMenuPath = null!;
 
+    [Export]
+    public NodePath? ExtinctionMessagePath;
```

Comment:
```suggestion
    public NodePath ExtinctionMessagePath = null!;
```

Looks like you aren't taking advantage of this being nullable so that's the correct way to define this.